### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by halicki**
Hey, this PR contains some style fixes to the Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # HOPS FLASK API
 
 # v.1.0
-### The root endpoint:
+
+## The root endpoint
```
Comment (jakub-skowron) [inline README.md:5]: asdf

File: README.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+
 `https://hopsflaskapi-production.up.railway.app`
 
-### Pagination:
+### Pagination
+
```
Comment (jakub-skowron) [inline README.md:10]: vhjcxz

Comment (halicki) [reply README.md:10]: Fixed

File: README.md (hunk 1 above)
Comment (halicki) [reply README.md:5]: I would not fix that because it is awesome!

**Pull request closed without merging**